tests/unit/Destroyable.ts

```diff
@@ -1,9 +1,8 @@
-import * as registerSuite from 'intern!object';
-import * as assert from 'intern/chai!assert';
+const { registerSuite } = intern.getInterface('object');
+const { assert } = intern.getPlugin('chai');
 import { Destroyable } from '../../src/Destroyable';
 
-registerSuite({
-	name: 'bases/Destroyable',
+registerSuite('bases/Destroyable', {
```

Comment:
do we need `bases` in the name now? Think this is a hangover from where they were moved from (compose)

Reply:
If you'd like me to update this as part of the PR, just give me the word

Reply:
as you're moving the name might be nice 👍